src/Http/Http.Extensions/gen/RequestDelegateGeneratorSources.cs

```diff
@@ -363,6 +363,100 @@ public void InvalidFormRequestBody(string parameterTypeName, string parameterNam
     }
 """;
 
+    public static string PropertyAsParameterInfoClass = """
+    file class PropertyAsParameterInfo : ParameterInfo
```

Comment:
```suggestion
    file sealed class PropertyAsParameterInfo : ParameterInfo
```

Reply:
This class is marked as file-private so the only classes that would be able to inherit from it are ones within the source generated file itself. I suppose it's valid to mark it as `sealed` to indicate to our own selves that we don't intend on inheriting from this class elsewhere in generated source but marking it as `sealed` here won't change anything for end users since it is already marked as file private.

Reply:
It's not about end user experience, it's about perf.
https://devblogs.microsoft.com/dotnet/performance-improvements-in-net-6/#peanut-butter

We turn on the analyzer to warn about sealing private/internal types so we get this mostly free perf https://github.com/dotnet/aspnetcore/blob/main/.editorconfig#LL199C16-L199C16
But it doesn't run on source generated code of course, so we must stay vigilant.